app/src/sharedTest/java/org/oppia/android/app/databinding/MarginBindingAdaptersTest.kt

```diff
@@ -292,6 +296,78 @@ class MarginBindingAdaptersTest {
     assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(40f)
   }
 
+  @Config(qualifiers = "port")
+  @Test
+  fun testMarginBindableAdapters_layoutParamsSettingPreservesMargins() {
+    val textView = activityRule.scenario.runWithActivity {
+      val textView: TextView = it.findViewById(R.id.test_margin_text_view)
+
+      // Set initial margins
+      setLayoutMarginStart(textView, /* marginStart= */ 24f)
+      setLayoutMarginEnd(textView, /* marginEnd= */ 40f)
+      setLayoutMarginTop(textView, /* marginTop= */ 16f)
+      setLayoutMarginBottom(textView, /* marginBottom= */ 32f)
+
+      return@runWithActivity textView
+    }
+
+    // Verify that the margins are correctly set after method calls
+    assertThat(textView.marginStart.toFloat()).isWithin(TOLERANCE).of(24f)
+    assertThat(textView.marginEnd.toFloat()).isWithin(TOLERANCE).of(40f)
+
+    val layoutParams = textView.layoutParams as ViewGroup.MarginLayoutParams
+    assertThat(layoutParams.topMargin.toFloat()).isWithin(TOLERANCE).of(16f)
+    assertThat(layoutParams.bottomMargin.toFloat()).isWithin(TOLERANCE).of(32f)
+  }
+
+  @Config(qualifiers = "land")
+  @Test
+  fun testMarginBindableAdapters_layoutParamsSettingPreservesMargins_landscape() {
```

Comment:
```suggestion
  fun testMarginBindableAdapters_landscapeMode_setLayoutParams_preservesMargins() {
```

Please apply naming pattern to the the tests below.

Reply:
Done.